### PR TITLE
Migrate AzureKeyVaultConfigStore to @configu/integrations

### PR DESCRIPTION
**Pull request by gagandeepp**
Closes https://github.com/configu/configu/issues/511

### CODE REVIEW

**Comment by gagandeepp** [comment]
@rannn505  Please review

File: packages/integrations/stores/azure-key-vault/src/azure-key-vault.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { DefaultAzureCredential, type DefaultAzureCredentialClientIdOptions } from '@azure/identity';
+import { SecretClient, type SecretClientOptions } from '@azure/keyvault-secrets';
+import { KeyValueConfigStore } from '@configu/ts';
```
Comment (RichardAkman) [inline packages/integrations/stores/azure-key-vault/src/azure-key-vault.ts:3]: Please use the `KeyValueConfigStore` from within the intergations package.
```suggestion
import { KeyValueConfigStore } from '@configu/integrations/src/utils/KeyValue';
```

Comment (gagandeepp) [reply packages/integrations/stores/azure-key-vault/src/azure-key-vault.ts:3]: Done

**Comment by gagandeepp** [comment]
@rannn505  Please review

**Comment by gagandeepp** [comment]
@davidmeirlevy please review

**Comment by RichardAkman** [comment]
@gagandeepp, seems like you resolved everything but one thing is missing. Please run `pnpm i` on the repository root folder to ensure the `pnpm-lock.yaml` is updated, this is a requirement to pass the CI.

**Review by RichardAkman — APPROVED** [review]
LGTM, I'll handle the update of the lock file.

**Pull request merged**